### ISSUE TITLE
is_snakecase("whatever_10") vs is_snakecase("whatever_1")

### ISSUE THREAD

**Issue by ZdenekM** [nficano/humps#128]
**Describe the bug**
According to humps, "whatever_10" is not a snake case.

**To Reproduce**
```python
>>> humps.is_snakecase("whatever_10")
False
>>> humps.is_snakecase("whatever_1")
True
```

**Expected behavior**
```python
>>> humps.is_snakecase("whatever_10")
True
>>> humps.is_snakecase("whatever_1")
True
```

**Desktop (please complete the following information):**
 - Version 1.6.1

**Comment by ronnie-llamado** [comment]
Closed by #167

**Comment by ZdenekM** [comment]
@ronnie-llamado Great! Is there going to be a new release?

**Comment by ronnie-llamado** [comment]
This fix is in the latest release: v3.0.2!

```python
>>> import humps
>>> humps.__version__
'3.0.2'
>>> humps.is_snakecase("whatever_10")
True
>>> humps.is_snakecase("whatever_1")
True
```

**Comment by ZdenekM** [comment]
Sorry, I confused the issue/problem with this one https://github.com/nficano/humps/issues/127.
